package.json

```diff
@@ -9,36 +9,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/balena-io/leviathan.git"
+    "url": "git+https://github.com/balena-os/leviathan.git"
   },
-  "author": "Theodor Gherzan <theodor@balena.io>",
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://github.com/balena-io/leviathan/issues"
+    "url": "https://github.com/balena-os/leviathan/issues"
   },
-  "homepage": "https://github.com/balena-io/leviathan",
+  "homepage": "https://github.com/balena-os/leviathan",
   "private": true,
-  "devDependencies": {
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.9.0",
-    "eslint-config-standard": "^14.1.0",
-    "eslint-plugin-import": "^2.19.1",
-    "eslint-plugin-jsdoc": "^20.0.5",
-    "eslint-plugin-lodash": "^6.0.0",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "husky": "^1.3.1",
-    "lint-staged": "^8.2.1",
-    "prettier": "^1.16.4",
-    "resin-lint": "^3.1.1"
```

Comment:
Can we keep a version of balena-lint in the dev dependencies for running on-demand?

Reply:
What do you mean? @klutchell The reason I removed it from the root is I don't understand why these dependencies are even there. I am pretty sure they are not being used and they are lining or prettifying anything in the root file.

Reply:
Maybe I am just asking the wrong question, but I like to be able to run `npm run balena-lint` in my project to clean it.
https://github.com/balena-io/balena-ci/blob/master/package.json#L33

Reply:
Right, you aren't asking the wrong question. In order to clean, prettify, or lint Leviathan. You can use the following commands 

`npm run prettify`
`npm run lint`

These will recursively run their respective commands in client, core, and worker. Do check them out and let me know if they are what you expect. With these you won't need global dependencies.

Also, if you ever need to run balena-lint anywhere even if it is installed or not installed you can use: `npx balena-lint --help`

Reply:
> npx balena-lint --help

TIL!